test/helpers/releases_helper_test.rb

```diff
@@ -15,42 +15,49 @@
     end
   end
 
-  describe "#github_commit_status_icon" do
+  describe "#commit_status_icon" do
     include ApplicationHelper
 
     it "renders an icon for success" do
-      html = github_commit_status_icon("success")
+      html = commit_status_icon("success")
       html.must_include "glyphicon-ok"
       html.must_include "text-success"
       html.must_include "Github status: success"
     end
 
     it "renders an icon for failure" do
-      html = github_commit_status_icon("failure")
+      html = commit_status_icon("failure")
       html.must_include "glyphicon-remove"
       html.must_include "text-danger"
       html.must_include "Github status: failure"
     end
 
     it "renders an icon for missing status" do
-      html = github_commit_status_icon("missing")
+      html = commit_status_icon("missing")
       html.must_include "glyphicon-minus"
       html.must_include "text-muted"
       html.must_include "Github status: missing"
     end
 
     it "renders an icon for pending status" do
-      html = github_commit_status_icon("pending")
+      html = commit_status_icon("pending")
       html.must_include "glyphicon-hourglass"
       html.must_include "text-primary"
       html.must_include "Github status: pending"
     end
 
-    it "renders an icon for pending status" do
-      html = github_commit_status_icon("pending")
-      html.must_include "glyphicon-hourglass"
-      html.must_include "text-primary"
-      html.must_include "Github status: pending"
+    it "renders an icon for error status" do
+      html = commit_status_icon("error")
+      html.must_include "glyphicon-exclamation-sign"
+      html.must_include "text-warning"
+      html.must_include "Github status: error"
+    end
+
+    it "assumes 'error' as default status" do
```

Comment:
might be more descriptive to do:
```suggestion
    it "assumes 'error' for 1-off states produced by release_status" do
```

Reply:
sure..